cmd/plotFunctions/plotFunctions.go

```diff
@@ -58,31 +58,78 @@ func usage() {
 			"Usage:\n" +
 			" plotFunctions functionKeyWord functionArgs leftBound rightBound steps outFile\n" +
 			"For AfsStationarity: defined by one parameter (called alpha). ex usage: plotFunctions AfsStationarity 0.001 0.001 0.999 1000 afsPlot.txt\n" +
+			"For AfsF: defined by two parameters (alpha and n: where alpha is the selection parameter and n is the total number of individuals. ex. Usage: AfsF 0.5,200 afsFPlot.txt\n" +
+			"For AfsProbability: defined by two parameters (alpha and n: where alpha is the selection parameter and n is the total number of individuals. ex. Usage: AfsProbability 0.5,200 afsProbabilityPlot.txt\n" +
 			"For Beta: defined by two parameters (called alpha and beta). ex usage: plotFunctions Beta 0.5,0.5 0.001 0.999 1000 betaPlot.txt\n" +
 			"For Gamma: defined by two parameters (called alpha and beta). ex usage: plotFunctions Gamma 0.5,0.5 0.001 0.999 1000 gammaPlot.txt\n" +
 			"For Normal: defined by two parameters (called mu and sigma). ex usage: plotFunctions Normal 0,0.5 -1 1 1000 normalPlot.txt\n" +
+			"For ChooseN: defined by 1 parameter (called n). Prints the binomial coefficient in logSpace. ex usage: plotFunctions ChooseN 40 chooseNPlot.txt\n" +
 			"options:\n")
 	flag.PrintDefaults()
 }
 
 func main() {
-	var expectedNumArgs int = 6
+	var expectedNumArgs int = 1
```

Comment:
Maybe just leave this uninitialized since we are not really using it, but setting it each time.  What happens when someone runs this with no arguments?  What does flag.Arg(0) on new line 77 evaluate to (give an error, nil)?  Maybe we should have the first "if" be a check for len(flag.Args()) being zero and then printing the usage?